### ISSUE TITLE
[rest] OpenAPI validations for nested objects

### ISSUE THREAD

**Issue by janrtvld** [hyperledger/aries-framework-javascript-ext#17]
OpenAPI validation is currently not working because it fails to validate nested objects. 

This can be resolved by implementing class-validator decorators in AFJ. https://github.com/hyperledger/aries-framework-javascript/pull/360

**Comment by brianorwhatever** [comment]
@janrtvld Is this why I am seeing a
```json
{
  ...,
  "constraints": {
      "isInstance": "each value in requestedPredicates must be an instance of ProofPredicateInfo",
      "isMap": "requestedPredicates must be a Map"
  }
}
```
error whenever I try to interact with the request proof endpoint? I have tried a number of things for the `requestedPredicates` and `requestedAttributes` fields but am not able to pass validation..

**Comment by TimoGlastra** [comment]
What is the payload you're sending to the rest API?

**Comment by TimoGlastra** [comment]
Is this fixed by #148?

**Comment by janrtvld** [comment]
Yes, we have removed the ones that aren't working (`Routing` & `Attachment`)
